tests/test_dialect_interfaces.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from xdsl.dialect_interfaces import OpAsmDialectInterface
+
+
+def test_op_asm_interface():
```

Comment:
Can you please also add a test for the general concept of a dialect interface? Probably with a test dialect interface defined in this file